operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md

```diff
@@ -96,18 +97,21 @@ The Release Manager \(Amy Blais\) owns the release announcement, including the f
 
 1. Highlight past successes: Cross link to promote traffic 
 2. Links tell a story: Links should indicate the content they lead to without clicking, avoid "read this blog post" and instead "learn more about the incident response alpha program"
+3. Actionable (e.g., taking steps 1-3, including links)
```

Comment:
In the context of a release announce, what is actionable? Perhaps an example may help.